src/main/resources/scss/pages/_dialogs.scss

```
@@ -1,3 +1,7 @@
 #toggle-switch-formsubmit {
     width: 100%;
-}
\ No newline at end of file
+}
+
+.jenkins-form-item-dialog-last {
```

new classes should be using `jdl` ( not possible to do this with a CSS selector though?)

```suggestion
.jdl-form-item-dialog-last {
```

---

fixed